django/contrib/admin/checks.py

```diff
@@ -19,9 +19,11 @@
 
 
 def check_admin_app(**kwargs):
-    from django.contrib.admin.sites import system_check_errors
-
-    return system_check_errors
+    from django.contrib.admin.sites import system_checks
```

Comment:
I'm not sure about that. Currently the check function (`check_admin_app`) is explicitly defined in the `checks` module ; this seems to be a good solution IMO. :wink: Why would it make more sense to change this?